.github/workflows/build-pr.yml

```diff
@@ -19,6 +19,12 @@ jobs:
         run: make bootstrap
       - name: run unit tests
         run: make test
+      - name: lint
```

Comment:
wdyt about running the linter before running unit tests?

Reply:
my bad! must have missed that, I will fix that